packages/angular/projects/ui-angular/src/lib/components/amplify-confirm-sign-up/amplify-confirm-sign-up.component.html

```diff
@@ -8,10 +8,11 @@
 <ng-template #confirmSignUpForm>
   <form data-amplify-form (submit)="onSubmit($event)" (input)="onInput($event)">
     <fieldset data-amplify-fieldset [disabled]="isPending">
-      <amplify-user-name-alias
```

Comment:
I'm scoping `amplify-user-name-alias` component usage to sign in and sign up only to keep other components clean from country code handling.

Reply:
Good idea!